s2spy/rgdr/rgdr.py

```diff
@@ -38,7 +38,7 @@ def spherical_area(latitude: float, dlat: float, dlon: float = None) -> float:
 
     lat = np.radians(latitude)
     h = np.sin(lat + dlat / 2) - np.sin(lat - dlat / 2)
-    spherical_area = h * dlon / np.pi * 4
+    spherical_area = h * dlon / np.pi / 4
```

Comment:
Minor comment, but to me writing the following is easier to comprehend:

spherical_area = h * dlon / (np.pi * 4)